tests/CreateFileFromStubTest.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+test('creates an action from stub', function () {
+    $fs = new \Illuminate\Filesystem\Filesystem();
+    $stub = 'action.stub';
+
+    $path = beyond_path() . "/tests/files/{$stub}";
+    beyond_copy_stub($stub, $path);
```

Comment:
We do not need to test if every stub is set as expected. I would test if `beyond_copy_stub` _does copy_ any stub as expected. Also we should test if all stubs we expect to be existing are existing.